### ISSUE TITLE
error while deploying org metadata in Scratch org

### ISSUE THREAD

**Issue by DigiPascal** [trailheadapps/az-insurance#85]
### Summary

I have been following step by step the instructions to deploy the sample app.
When I launch the command to deploy SF orga metadata I get a "Unexpected file found in package directory"

I am using the latest sfdx-cli.
(sfdx-cli/7.150.0 darwin-x64 node-v16.15.0)

### Salesforce Org Type

Scratch Org

### Steps To Reproduce

sfdx auth:web:login -d -a myhuborg
git clone https://github.com/trailheadapps/az-insurance
cd az-insurance
sfdx force:org:create -s -f config/project-scratch-def.json -a az-insurance
sfdx force:community:create --name "Dummy" --templatename "Aloha" -p "dummy"
**sfdx force:source:deploy -m ApexClass,Layout,CustomObject,LightningComponentBundle,ManagedContentType,CustomObject,StaticResource,CustomTab,PermissionSet,Flow**

### Current Behavior

Error message :

az-insurance % sfdx force:source:deploy -m ApexClass,Layout,CustomObject,LightningComponentBundle,ManagedContentType,CustomObject,StaticResource,CustomTab,PermissionSet,Flow
ERROR running force:source:deploy:  Unexpected file found in package directory: /Users/Pascal/Code/az-insurance/force-app/main/default/managedContentTypes/cms_actionimage.managedContentType-meta.xml

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by muenzpraeger** [comment]
Thanks for reaching out. Can you please share the output of the command `sfdx plugins --core`? My first guess is that you have an outdated or manual dependency in the CLI that is causing this.

For comparison, here is my output:

```zsh
az-insurance on  main via ⬢ v14.17.5 took 57s with org az-insurance1
➜ sfdx plugins --core
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.3.1 (core)
@oclif/plugin-not-found 1.2.6 (core)
@oclif/plugin-plugins 1.10.11 (core)
@oclif/plugin-update 1.5.0 (core)
@oclif/plugin-warn-if-update-available 1.7.3 (core)
@oclif/plugin-which 1.0.4 (core)
@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
alias 2.0.0 (core)
apex 0.11.0 (core)
auth 2.0.2 (core)
community 1.1.4 (core)
config 1.4.6 (core)
custom-metadata 1.1.0 (core)
data 0.6.15 (core)
generator 2.0.0 (core)
info 2.0.0 (core)
limits 2.0.0 (core)
org 1.12.1 (core)
salesforce-alm 54.3.0 (core)
schema 2.1.0 (core)
sfdx-cli 7.150.0 (core)
signups 1.0.0 (core)
source 1.9.7 (core)
telemetry 1.4.0 (core)
templates 54.6.0 (core)
trust 1.1.0 (core)
user 1.7.1 (core)
```

**Comment by DigiPascal** [comment]
Hi René,
Thanks for the quick feedback. You saw it right !
There was indeed a difference in the salesforce-alm version, I had updated to your version and the metadata now upload to the scratch org. I could proceed to installing the project.

I am closing the issue

Thanks,
Again
